specviz/app.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, *args, file_path=None, file_loader=None, embedded=False,
         self.load_local_plugins()
 
         # cache the line lists for speedier access
-        from .core import linelist
+        from .plugins.line_labels import linelist
```

Comment:
This should not be explicitly imported. Plugins defined with the decorator will automatically be imported into the namespace.

Reply:
Done.